apps/src/dance/lab2/views/DanceView.tsx

```diff
@@ -64,28 +142,57 @@ const DanceView: React.FunctionComponent<
     );
   }, [levelProperties, initialSources, dispatch]);
 
-  const userType = useAppSelector(state => state.currentUser.userType);
-  const under13 = useAppSelector(state => state.currentUser.under13);
-  const [filterOn, setFilterOn] = useState<boolean>(
-    getFilterStatus(userType, under13)
-  );
-  const turnOffFilter = useCallback(() => setFilterOn(false), []);
+  useEffect(() => {
+    const {isProjectLevel, freePlay, customHelperLibrary, validationCode} =
+      levelProperties;
+    // record a replay log (and generate a video) for both project levels and any
+    // course levels that have sharing enabled
```

Comment:
Video might no longer happen?

Reply:
Yeah, sounds like it. I'll look at removing/hiding this as necessary